msupply-foundation/open-msupply
# 5601 add feedback form inputs

Fixes #5601 

# 👩🏻‍💻 What does this PR do?

- [ ] Add a new section to the help page accordian, accessible to all users, called `Contact us`
- [ ] Inside it, we should have a subheader `Feedback form` and then the two inputs for `email` and `message`
- [ ] Add a `useFeedbackForm` hook, which manages the draft state for the feedback form
- [ ] Add a `Send` button (can use [mail](https://feathericons.com/?query=mail) icon) which `console.log`s the feedback form, and shows a `success` snack that the feedback form has been saved

This is to add a form to the help page so that a user can contact us.
This is part of fulfilling the DFAT GEDSI project's objectives.

This is what the updated help page looks like (click to enlarge):

<img width="720" alt="Screenshot 2024-12-03 at 11 06 14" src="https://github.com/user-attachments/assets/40d3141c-5b7f-4835-9ea8-bd144c1bcd38">

## 💌 Any notes for the reviewer?

- Is the UI in accordance with our guidelines (especially the mail icon)?
- Is the file structure (found in `/client>pakages>host>src>Help`) in accordance with our guidelines?
- Is the styling code (specifically the way the feedback form's components are wrapped in <DetailContainer><Grid> tag, and the mail icon's file in /client/packages/common/src/ui/icons/Mail.tsx) in accordance with our guidelines?

# 🧪 Testing



1. Open the help page
2. Enter stuff in the forms
3. Click the send button
4. Do the above at different screen sizes
5. See if it is rendering and behaving as expected at all those different sizes

# 📃 Documentation

**Part of an epic**: documentation will be completed for the feature as a whole
